supervisord.conf

```diff
@@ -42,7 +42,7 @@ stderr_logfile_maxbytes=0
 # your custom programs go here
 #Server
 [program:server]
-command= /app/serverlessrust
+command= /app/oyster-serverless
```

Comment:
private key as argument?

Reply:
This file is not updated to the latest settings, including the `setup.sh`. Will update this in that PR, as it will have further changes related to this.